### PR TITLE
Small updates for input component, text fixes

### PR DESCRIPTION
**Pull request by macandcheese**
Fixes incrementing number type input with appended buttons
Removes custom textarea resize handle in ie11
Fixes display of custom textarea resize handle in Firefox
Updates style of incrementing number buttons
Fixes tests for input

### CODE REVIEW

File: src/components/calcite-input/calcite-input.tsx [hunk 1]
```diff
@@ -47,7 +48,16 @@ export class CalciteInput {
   @Prop({ mutable: true, reflect: true }) alignment: "start" | "end" = "start";
 
   /** input value */
-  @Prop({ mutable: true }) value?: string | null = "";
+  @Prop({ mutable: true, reflect: true }) value?: string = "";
+
+  /** input step */
+  @Prop({ mutable: true, reflect: true }) step?: string = "";
```
Comment (paulcpederson) [inline src/components/calcite-input/calcite-input.tsx:54]: do these actually need to be mutable? Are you changing them from inside this component?

File: src/components/calcite-input/calcite-input.tsx [hunk 2]
```diff
@@ -308,6 +321,21 @@ export class CalciteInput {
...
+  //--------------------------------------------------------------------------
+  //
+  //  Public Methods
+  //
+  //--------------------------------------------------------------------------
+
+  /** focus the rendered child element */
+  @Method()
+  async setFocus() {
+    if (!this.childEl) {
```
Comment (paulcpederson) [inline src/components/calcite-input/calcite-input.tsx:333]: can you use `this.childEl?.focus()` instead of ifs here?

**Comment by paulcpederson** [comment]
looks good, couple of questions but feel free to merge

**Comment by macandcheese** [comment]
Addressed comments and merged 🤖

**Pull request merged**